build.sbt

```diff
@@ -641,7 +640,6 @@ lazy val unit = project
       "ch.epfl.scala" %% "bloop-config" % V.bloop,
       "org.scalameta" %% "munit" % V.munit,
       // The dependencies listed below are only listed so Scala Steward
-      // will pick them up and update them. They aren't actually used.
```

Comment:
I think this might have been removed by mistake?

Reply:
Ah, yes, fixed now.